src/component/MyProfile.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+import React from "react";
+import UpdateRocket from "./RocketProfile";
+import Missionprofile from "./missionProfile";
+
+
+const MyProfile = () => {
+
+  return(
+    <table className="profile_d">
+      <Missionprofile />
+      <UpdateRocket />
+    </table>
```

Comment:
- [ **OPTIONAL** ] What do you think if the `My Profile` section can have a placeholder default message when the "My Missions" or "My Rockets" lists are empty (**no missions joined or no rockets reserved**) to make them look meaningful and better.